client/extensions/woocommerce/components/form-location-select/states.js

```diff
@@ -75,7 +75,7 @@ class FormStateSelectFromApi extends Component {
 	renderDisabled = () => {
 		const { translate } = this.props;
 		return (
-			<FormSelect disabled>
+			<FormSelect autoComplete="off" disabled>
```

Comment:
curious why the autoComplete off here?

Reply:
It avoids a Chrome warning about auto-complete and disabled